sympy/utilities/misc.py

```diff
@@ -415,7 +415,7 @@ def translate(s, a, b=None, c=None):
         s = replace(s, mr)
         table = maketrans(a, b)
         # s may have become unicode which uses the py3 syntax for translate
-        if type(table) is str and type(s) is str:
```

Comment:
the inline here indicates that the code knows about the unicode possibility and the code should not be changed